docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -43,13 +43,26 @@ Does this look interesting? Read on!
 
 NOTE: the `list()` method might be surprising at first. It takes fragments of HQL (JP-QL) queries and contextualizes the rest. That makes for very concise but yet readable code.
 
+NOTE: what you just see is the link:https://www.martinfowler.com/eaaCatalog/activeRecord.html[active record pattern], sometimes just called the entity pattern.
+Hibernate with Panache also provides the more classical link:https://martinfowler.com/eaaCatalog/repository.html[repository pattern] via `PanacheRepository`.
```

Comment:
```suggestion
Hibernate with Panache also for the use of the more classical link:https://martinfowler.com/eaaCatalog/repository.html[repository pattern] via `PanacheRepository`.
```

Reply:
This suggestion look like a verb is missing ... Di you means `allow for the use` ?

Reply:
Yup :)